packages/storybook/cxl-ui/cxl-app-layout/layout=2c-r.stories.js

```diff
@@ -370,6 +366,49 @@ export const CXLAppLayout2cr = () => {
           <p>Ending paragraph here.</p>
         </div>
       </article>
+
+      <div slot="action-bar">
+        <!--
+        <vaadin-button
+          theme="contrast"
+          onclick="window.location.href='https://conversionxli.warmpress.com/lesson/persuasion-slide-introduction/'"
+          tabindex="0"
+          role="button"
+        >Prev<iron-icon slot="prefix" icon="lumo:arrow-left"></iron-icon
+        ></vaadin-button>
+        <vaadin-button
+          theme="secondary"
+          onclick="window.location.href='https://conversionxli.warmpress.com/lesson/3-persuasion-slide/'"
+          tabindex="0"
+          role="button"
+        >Next<iron-icon slot="suffix" icon="lumo:arrow-right"></iron-icon
+        ></vaadin-button>
+        -->
```

Comment:
@saas786 can you remove this commented out code?

Reply:
@pawelkmpt 
Ok, done, removed the commented code.